language.py

```diff
@@ -1498,6 +1498,8 @@ def to_lean_binop(bop, state):
   bitwidth = unify_bitwidths([lv1.expr.bw(), lv2.expr.bw()])
   propagate_bitwidth(lv1.expr, bitwidth)
   propagate_bitwidth(lv2.expr, bitwidth)
+  if bop.flags != []:
```

Comment:
```suggestion
  if bop.flags:
```

Reply:
that sounds like a bad idea, no? it makes the code less readable/more opaque